### ISSUE TITLE
IF adding a ride from a group

### ISSUE THREAD

**Issue by MattiasJay** [Skjutsgruppen/skjutsgruppen-reactnative#213]
IF participant is in her or his group and wants to add a ride, they are forwarded to the same process as the add section but with a few changes.

1. They can start to write a comment in the group and then press "Offer a ride" or "Ask for a ride" or "comment", see this slide: https://projects.invisionapp.com/d/main#/console/12489615/274003368/preview

2. IF they choose to offer or ask for they are forwarded to due process in the add section with these changes:

- [x] Comment from above comment section in the group is prefilled in the text box. This can be changed.
- [x] "From" and "Where" is pre filled from the specifics of the group, IF the group has a specific stretch. Same functionality as already is in current groups, if you want to copy that code. This can be changed.
- [x] Remember to prefill the stops along the way as well, if this applies.
- [x] On the last step, share and publish, the group in which the process was started (for instance "The Metal group") is pre-checked and can *not* be unchecked. It is to be prechecked with the grey check in the design (same as is used for "share with the movement").

3. Lastly, two different designs for if return ride or not return ride:

- [x] IF return ride: https://projects.invisionapp.com/d/main#/console/12489615/276582218/preview

- [x] IF no return ride: https://projects.invisionapp.com/d/main#/console/12489615/276582682/preview

**Comment by MattiasJay** [comment]
Wrong texts and wrong how it looks, both rides and groups @anjhero @sachit-singh .

Ride added in group: It should not be "(" marks around the group name, but as I tried to show above the name of the group should be bold. Make it bold and blue an as a link to the group. 

I also wrote that everything in italic should be blue and as a link. That means that "Read more about our group here" (see the comment above from Match 28th) should be blue and as a link. 

Ride added in ride: this is even more wrong, look on my comment from March 30th above how it should look like @sachit-singh
